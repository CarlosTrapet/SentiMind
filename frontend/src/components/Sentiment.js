import React from 'react';

const Sentiment = props => {
  return <h2>{props.sentiment}</h2>;
};

export default Sentiment;
