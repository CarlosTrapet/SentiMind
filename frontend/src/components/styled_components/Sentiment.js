import styled from 'styled-components';

const StyledSentiment = styled.h2`
  color: white;
  font-size: 30px;
`;

export default StyledSentiment;
