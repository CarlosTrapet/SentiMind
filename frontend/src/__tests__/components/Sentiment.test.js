import React from 'react';
import {shallow} from 'enzyme';

import Sentiment from '../../components/Sentiment';

describe('Sentiment', () => {
  const sentiment = shallow(<Sentiment/>)

  it('renders correctly', () => {
    expect(sentiment).toMatchSnapshot();
  })
});
