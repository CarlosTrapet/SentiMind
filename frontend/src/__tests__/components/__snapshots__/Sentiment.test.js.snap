// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sentiment renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Sentiment
    sentiment="negative"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": " Positive, Negative, Neutral ",
    },
    "ref": null,
    "rendered": " Positive, Negative, Neutral ",
    "type": "h2",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " Positive, Negative, Neutral ",
      },
      "ref": null,
      "rendered": " Positive, Negative, Neutral ",
      "type": "h2",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
